app/views/layouts/application.html.erb

```diff
@@ -39,54 +40,55 @@
               // Invoke the plugin
               $('input, textarea').placeholder();
               $('input, iframe').placeholder();
-          });    
+          });
         </script>
-    <![endif]-->  
+    <![endif]-->
 
     <%= yield(:head) %>
     <%= csrf_meta_tags %>
 
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        
+
   </head>
-  <body>
+  <body style="margin-bottom:75px;">
     <!--p class="alert alert-danger">The DMPRoadmap service will be down from 1600 BST (UTC+1) on Friday, 15th May 2015 to allow essential updates.  Please ensure you have saved your data and logged out before this time to avoid any problems.  We will be back again as soon as possible on Friday evening.</p-->
       <!--p class="alert alert-success">Our essential updates have been completed, and DMPRoadmap is up and running again. Many thanks for your patience.</p -->
 
-    <div class="container">
-
       <!-- header renedering -->
-      <header class="row">
-        <%= render "layouts/header" %>
+      <header>
+        <%= render partial: "layouts/header" %>
       </header>
 
-      <% 
+      <%
       has_alert = (alert || flash[:alert] || flash[:error])
       has_notice = (notice || flash[:notice])
       %>
 
-      <div id="notification-area"
-           class="roadmap-<%= has_alert ? 'alert' : 'info' %>-box" 
-           style="visibility: <%= (has_alert || has_notice) ? 'visible' : 'hidden' %>;"<%= (has_notice ? ' role=status' : (has_alert ? ' role=alert' : '')) %>>
-        <i class="fa <%= has_alert ? ' fa-times-circle' : ' fa-check-circle' %>" aria-hidden="true"></i>
-        <span class="aria-only"><%= has_alert ? _('Error:') : _('Notice:') %></span>
-        <span><%= raw (has_alert ? alert : notice) %></span>
-      </div>
-
-      <main>
+      <main class="container" style="<% if !user_signed_in? or current_user.org.nil? %>padding-top:75px;<% end %>">
+        <div id="notification-area"
+             class="roadmap-<%= has_alert ? 'alert' : 'info' %>-box"
+             style="visibility: <%= (has_alert || has_notice) ? 'visible' : 'hidden' %>;"<%= (has_notice ? ' role=status' : (has_alert ? ' role=alert' : '')) %>>
```

Comment:
you should be able to use bootstrap classes here instead of setting visibility manually:

https://getbootstrap.com/docs/3.3/css/#helper-classes-show-hide